spec/javascripts/components/layout-super-navigation-header-spec.js

```diff
@@ -735,8 +738,14 @@ describe('The super header navigation', function () {
           expect($button).toHaveAttr('aria-label', hideLabel)
         })
 
-        it('updates the button’s visual state', function () {
-          expect($button).toHaveClass('gem-c-layout-super-navigation-header__open-button')
+        it('triggers the correct google analytics custom event', function () {
+          expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith('headerClicked', 'testingOpened', {
+            label: 'none'
+          })
+        })
+
+        xit('updates the button’s state', function () {
+          expect($button).toHaveClass('govuk-header__menu-button--open')
         })
```

Comment:
This skipped test can be removed; it's been updated with:

https://github.com/alphagov/govuk_publishing_components/blob/d4a897b33089817b476fec645cf11f43d2e5bc93/spec/javascripts/components/layout-super-navigation-header-spec.js#L738-L740

```suggestion
```